### ISSUE TITLE
Projects will not run, due to not finding a library.

### ISSUE THREAD

**Issue by miguelperez** [NativeScript/playground-feedback#80]
Hi, I am having some errors when trying to run any project in the playgroud. Any sample or the initial project will fail with an error due to not loading nativescript-vue.

### Tell us about the problem
Please, ensure your title is less than 63 characters long and starts with a capital
letter.

### Did you try it outside of {N} Playground?
- No

### Which browser are you using?
- Chrome
- Safari

### Are there any errors/warnings logged in your browser console?
No

### Which platform(s) does your issue occur on?
- iOS

### Are there any entries logged in the Device Logs tab?
yes
```
[iPhone X]: 2018-10-01 08:15:02.587 nsplaydev[4247:899101] ***** Fatal JavaScript exception - application has been terminated. *****
2018-10-01 08:15:02.587 nsplaydev[4247:899101] Native stack trace:
1 0x10520f73c NativeScript::reportFatalErrorBeforeShutdown(JSC::ExecState*, JSC::Exception*, bool, bool)
2 0x105240068 -[TNSRuntime executeModule:referredBy:]
3 0x104e74714 -[UIApplication(PlayLiveSync) swizzled_executeModule:]
4 0x1049413dc
5 0x186d5cb94 
2018-10-01 08:15:02.588 nsplaydev[4247:899101] JavaScript stack trace:
2018-10-01 08:15:02.588 nsplaydev[4247:899101] 1 require@file:///app/app.js:3:31
2 anonymous@file:///app/app.js:3:31
3 evaluate@[native code]
4 moduleEvaluation@[native code]
5 @[native code]
6 promiseReactionJob@[native code]
2018-10-01 08:15:02.588 nsplaydev[4247:899101] JavaScript error:
file:///app/app.js:3:31: JS ERROR Error: Could not find module 'nativescript-vue'. Computed path '/var/mobile/Containers/Data/Application/F6487772-76CA-4226-BA1D-767A604B7A8B/Documents/Playground/LiveSync/app/tns_modules/nativescript-vue'.
2018-10-01 08:15:02.588 nsplaydev[4247:899101] PlayLiveSync: Uncaught Exception
2018-10-01 08:15:02.589 nsplaydev[4247:899101] *** JavaScript call stack:
(

)
2018-10-01 08:15:02.589 nsplaydev[4247:899101] *** Terminating app due to uncaught exception 'NativeScript encountered a fatal error: Error: Could not find module 'nativescript-vue'. Computed path '/var/mobile/Containers/Data/Application/F6487772-76CA-4226-BA1D-767A604B7A8B/Documents/Playground/LiveSync/app/tns_modules/nativescript-vue'.
at 
1 require@file:///app/app.js:3:31
2 anonymous@file:///app/app.js:3:31
3 evaluate@[native code]
4 moduleEvaluation@[native code]
5 @[native code]
6 promiseReactionJob@[native code]
', reason: '(null)'
*** First throw call stack:
(0x187313ef8 0x1864e1a40 0x10520fe60 0x105240068 0x104e74714 0x1049413dc 0x186d5cb94)
libc++abi.dylib: terminating with uncaught exception of type NSException
2018-10-01 08:15:02.589 nsplaydev[4247:899101] PlayLiveSync: Uncaught Exception

```
### Please provide the following information shown in devices tab in the playground:
- Model
iphone x
- OS version
12
- Preview app version
1.14.0
- Runtime version
4.2.0

Device Name | Model | OS version | Preview app version | Runtime version
-- | -- | -- | -- | --
iPhone X | iPhone X | iOS 12.0.0 | 1.14.0 | 4.2.0

### Please tell us how to recreate the issue with as much detail as possible.
I am just trying to run any of the samples. All of them fail due to some library missing.

**Comment by vhristov5555** [comment]
Could you please send us the url you are trying to open ?
I have try with several and they are working on my side.
Could you please not close the app and scan again ?

**Comment by miguelperez** [comment]
Hi @vhristov5555 I forgot to mention here that it suddenly started working again yesterday.
I will close this issue and will re-open it if I find it happening again on my end. 
Thanks for your reply ;)

**Comment by vhristov5555** [comment]
I am happy that you were able to see the samples in action. If you have further problems please write to us.

**Comment by MarikVal** [comment]
Got the same problem today - can't preview any project with {N} preview app. 
> file:///app/app.js:3:31: JS ERROR Error: Could not find module 'nativescript-vue'. Computed path '/var/mobile/Containers/Data/Application/B429F5BD-E387-4ACE-A7B4-13B99D60095B/Documents/Playground/LiveSync/app/tns_modules/nativescript-vue'.

**Comment by mr-simonski** [comment]
I got the same problem for a couple of days on my iPhone 7 with iOS 13.1.2. 

When starting any code in the preview mode the mentioned error occured: Error: Could not find module 'nativescript-vue'

I tried to close the running applications (both preview app and playground) but this didn't help. 

My solution was to uninstall both apps and reinstall them - after that my app works in preview mode as expected.
